framework/auth/views.py

```diff
@@ -88,11 +88,10 @@ def forgot_password_post():
                 reset_link=reset_link
             )
         status.push_status_message(
-            'An email with instructions on how to reset the password for the '
-            'account associated with {0} has been sent. If you do not receive '
-            'an email and believe you should have please '
-            'contact OSF Support.'.format(email)
-        )
+            ('An email with instructions on how to reset the password' +
```

Comment:
You are missing spaces at the ends of these words.

Also, the `+` signs are unnecessary. See https://cosdev.readthedocs.org/en/latest/style_guides/python.html#line-lengths . 

The previous formatting of this message was correct; can you revert this change and just add the `'success'` argument?